src/app/shared/components/list/list.component.ts

```diff
@@ -1,6 +1,6 @@
 import { Component, Input, OnInit } from '@angular/core';
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
-import { NgComponentOutlet } from '@angular/common';
+// import { NgComponentOutlet } from '@angular/common';
```

Comment:
This is weird it's not throwing an error. I thought I had to import this to use the ngComponentOutlet in the template, but it seems to be working.

Reply:
Yep, it's not being used which like you said is weird. but works haha